SDKS-4947 Update README files (#187)
ForgeRock/ping-android-sdk: recaptcha-enterprise/README.md

# ReCaptcha Enterprise Module

> Seamless integration of Google ReCaptcha Enterprise into Ping Identity's Journey workflows for Android.

Reviewer: ```suggestion
Seamless integration of Google reCAPTCHA Enterprise into authentication journeys for Android.
```

Reviewer: Minor I know, but they capitalize it "reCAPTCHA".

---

## Overview

> The **ReCaptcha Enterprise** module provides seamless integration of Google ReCaptcha Enterprise verification into Ping Identity's Journey authentication flows. This powerful module enables developers to add advanced bot detection and abuse protection to their Android applications with minimal configuration.

Reviewer: ```suggestion
The **reCAPTCHA Enterprise** module provides seamless integration of Google reCAPTCHA Enterprise verification into authentication journeys. This powerful module enables you to add advanced bot detection and abuse protection to your Android applications with minimal configuration.
```
